capstone-api/models/User.js

```diff
@@ -222,6 +220,21 @@ class User {
       groupInfo;
     let group = new Parse.Object("Group");
 
+    if (groupName === "") {
+      return "Name cannot be empty!";
+    }
+
+    let uniqueNameQuery = new Parse.Query("Group");
```

Comment:
also something you can do to refactor in a later diff, if the variable doesn't get changed or you don't intend on it to change, i recommend using const instead of let.

Reply:
will change for all components in later PR